evasquez26/angular4-startup
# Left panel navigation: profile, schools and courses

## Story / Task

[ONE-92](https://bixlabs.atlassian.net/browse/ONE-92), [ONE-93](https://bixlabs.atlassian.net/browse/ONE-93), [ONE-164](https://bixlabs.atlassian.net/browse/ONE-164) and [ONE-165](https://bixlabs.atlassian.net/browse/ONE-165)

## What this PR does?

Links the corresponding menu options to the corresponding pages.


### Before 

Left panel options did not take anywhere

### After

Left panel options takes to their corresponding page

### How to test / reproduce

- Run the app
- Observe that clicking the options on the left panel shows a different content

### TODOs

N/A

### Related PRs

N/A

## Migrations

NO

Thanks!
